final_task/rss_reader/functions/check_func.py

```diff
@@ -0,0 +1,54 @@
+"""Contain checking functions"""
+
+import requests
+import logging
+import sys
+import classes.exceptions as exc
+
+def check_internet_connection(logger):
+    """Check if network connection is avaliable"""
+    try:
+        logger.info("Checking internet connection...")
+        requests.get('https://www.google.com/', timeout=1)
+        return True
+    except requests.ConnectionError:
+        raise exc.InternetConnectionError("No internet access")
```

Comment:
Why would you want this function to raise exception? Following it's name, it should return either True or False

Reply:
check_internet_connection() function invokes in main() function and should stop work of program, if we retrieve rss from internet source. Well, in convertation.py file, it invokes to decide should we insert images links or not, and in this case I had to handle exception InternetConnectionError. I didn't changed check_internet_connection() function, because it would make main() function more complicated.